tests/integration/service_test.py

```diff
@@ -397,7 +397,7 @@ def test_execute_convergence_plan_when_host_volume_is_removed(self):
 
         assert not mock_log.warn.called
         assert (
-            [mount['Destination'] for mount in new_container.get('Mounts')],
+            [mount['Destination'] for mount in new_container.get('Mounts')] ==
```

Comment:
This has now been fixed in #3619, but if GH isn't complaining, I guess the merge won't blow up.